FinalProject/gs-rest-service-master/complete/src/main/java/bo/Product.java

```diff
@@ -0,0 +1,50 @@
+package bo;
+
+public class Product {
+	private int productId;
+	private String productName;
+	public String category;

```

Comment:
Minor for this project: It's not necessary but for better design category could be a class too.